docs/ansible/roles/nslcd/getting-started.rst

```diff
@@ -21,9 +21,6 @@ documentation for more details):
 
 .. code-block:: none
 
-   [debops_service_ldap]
-   hostname
-
```

Comment:
This was added intentionally and it should stay. Using the `nslcd` role by itself is fine, but if you also add the `ldap` role, `nslcd` will be automatically integrated with the rest of the LDAP environment managed by DebOps.

Reply:
The requested change is done.